x-pack/plugins/infra/public/components/alerting/logs/expression_editor/criterion.tsx

```diff
@@ -0,0 +1,265 @@
+              ownFocus
+              panelPaddingSize="s"
+              anchorPosition="downLeft"
+            >
+              <div>
+                <EuiPopoverTitle>{criterionFieldTitle}</EuiPopoverTitle>
+                <EuiFormRow isInvalid={errors.field.length > 0} error={errors.field}>
+                  <EuiSelect
+                    compressed
+                    value={criterion.field}
+                    onChange={handleFieldChange}
+                    options={fieldOptions}
+                  />
+                </EuiFormRow>
+              </div>
+            </EuiPopover>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiPopover
+              id="criterion-comparator-value"
+              button={
+                <EuiExpression
+                  description={criterion.comparator}
+                  uppercase={true}
+                  value={criterion.value}
+                  isActive={isComparatorPopoverOpen}
+                  color={
+                    errors.comparator.length === 0 && errors.value.length === 0
+                      ? 'secondary'
+                      : 'danger'
+                  }
+                  onClick={() => setIsComparatorPopoverOpen(true)}
+                />
+              }
+              isOpen={isComparatorPopoverOpen}
+              closePopover={() => setIsComparatorPopoverOpen(false)}
+              ownFocus
+              panelPaddingSize="s"
+              anchorPosition="downLeft"
+            >
+              <div>
+                <EuiPopoverTitle>{criterionComparatorValueTitle}</EuiPopoverTitle>
+                <EuiFlexGroup gutterSize="l">
+                  <EuiFlexItem grow={false}>
+                    <EuiFormRow isInvalid={errors.comparator.length > 0} error={errors.comparator}>
+                      <EuiSelect
+                        compressed
+                        value={criterion.comparator}
+                        onChange={e =>
+                          updateCriterion(idx, { comparator: e.target.value as Comparator })
+                        }
+                        options={compatibleComparatorOptions}
+                      />
+                    </EuiFormRow>
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiFormRow isInvalid={errors.value.length > 0} error={errors.value}>
+                      {fieldInfo?.type === 'number' ? (
+                        <EuiFieldNumber
+                          compressed
+                          value={criterion.value as number | undefined}
+                          onChange={e => {
+                            const number = parseInt(e.target.value, 10);
+                            updateCriterion(idx, { value: number ? number : undefined });
+                          }}
+                        />
+                      ) : (
+                        <EuiFieldText
```

Comment:
How about we add some double quotes around the `<EuiFieldText>` to signify that it's a string value instead of a number?

<img width="480" alt="Screen Shot 2020-04-23 at 12 11 50 PM" src="https://user-images.githubusercontent.com/1445834/80128822-d88f2780-855b-11ea-9120-60abd1e966ea.png">

Reply:
Hmm, I think this could potentially mislead / frustrate people. I understand the intention, but I can also see people entering `error` and then seeing `"error"` and thinking "huh, I haven't added quotations". 

Or, if there's a scenario where people actually want to have the quotations as part of the value they're matching, we either need to display double quotes which is messy, or not render them if the user has used quotes in the value. This again leads to a misleading situation. 

I think it might be best to leave this one? Or defer to design for the next iteration?

Reply:
Okay we can leave this. Changing `EQUALS` to `IS` should be good enough.